src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -80,7 +81,7 @@ class DraftAdjudicationService(
       )
     }.toMutableList()
     if (draftAdjudication.offenceDetails != null) {
-      draftAdjudication.offenceDetails!!.removeAll(draftAdjudication.offenceDetails!!)
+      draftAdjudication.offenceDetails!!.clear()
```

Comment:
Out of interest do we need the double !! if we've checked null? is Kotlin smart enough it can't be null?